### ISSUE TITLE
Users do not understand what is CSV fail

### ISSUE THREAD

**Issue by anettlinno** [citizenos/citizenos-fe-old#668]
**What is the user story?**
Users who download the results file after ID voting gets two options: 1) container without CSV fail 2) container with CSV fail. They don´t understand what is CVS fail. 

<img width="514" alt="T6lge" src="https://user-images.githubusercontent.com/45175679/112165059-11fe6600-8bf7-11eb-9d08-196df6069e58.png">

**Which feature needs to be updated and how?**
Adding information bubble (question mark or exclamation) with the description about what is CSV fail would help users to understand what it is. 

**Additional information.**
N/A

**Comment by BeccaMelhuish** [comment]
My suggestion is that we have the help bubble where I've highlighted it in yellow, and that we include the following text so that it covers the FAQ about which file to choose for submitting to which institution: 


Submitting to the Commercial Register? Choose “Container without CSV”. 

Submitting to the Riigikogu or local government? Choose “Container with CSV”. 

A CSV file is data presented in a tabular format which can be opened in any spreadsheet programme, such as Excel or Google Sheets. 

Need more info? View Q4 and Q5 in our FAQs (hyperlinked 'FAQs')   



![image](https://user-images.githubusercontent.com/61320526/116081642-ce39c900-a69a-11eb-95ef-dc0402062d3a.png)

**Comment by loorm** [comment]
Triage 36. @helenkask Let's just change the naming of the menu items. 
"Container without CSV" should become "Machine-readable file"
"Container with CSV" should become "Human-readable file"

**Comment by BeccaMelhuish** [comment]
We (me, @tiblu @ilmartyrk, in meeting on 06.05) discussed this and agreed that adding the Help Button would be the best solution to help the user understand two of the identified FAQs (4 and 5), but that @kevincrepin could provide input as to where the help button should go (in the place I marked in yellow above, or next to the drop down button?) and provide the pop-up design as per issue #693. We discussed the "human-readable / machine-readable file" idea but felt this would cause confusion too. 

We also discussed that the text within the Help Button pop-up should be edited to include the Estonian words for the relevant organisations (as only Estonians will currently land on this page), and therefore be as follows. (Sorry, didn't catch the Estonian!):

Submitting to the (insert Estonian word for "Commercial Register")? Choose “Container without CSV”.

Submitting to the Riigikogu or (insert Estonian word for "local government")? Choose “Container with CSV”.

A CSV file is data presented in a tabular format which can be opened in any spreadsheet programme, such as Excel or Google Sheets.

Need more info? View Q4 and Q5 in our FAQs (hyperlinked 'FAQs')


@ilmartyrk I haven't included the text about attachments not being included in the folder here as it felt a bit out of place, but will include it in the linked FAQ which people can visit if they need more info, where it lists everything that is included in the BDOC folders.

@kevincrepin your thoughts on all this from a UX perspective very welcome!

**Comment by tiblu** [comment]
@BeccaMelhuish Are we waiting for @kevincrepin -s input or there is something we are to do right away? 
IF we are waiting, we will move it to "In preparation".

**Comment by tiblu** [comment]
As a quick fix for Estonian users, I changed Estonian translations:

![image](https://user-images.githubusercontent.com/7405383/118463994-0a47d300-b709-11eb-9a02-24f7054f8492.png)

**Comment by tiblu** [comment]
@anettlinno @BeccaMelhuish moved back to "In preparation" as there is not enough information to work on this.

**Comment by loorm** [comment]
No design changes needed. Let's change the menu names.
"Container without CSV" becomes "Machine-readable file"
"Container with CSV" becomes "Human-readable file"

For Estonian translations:
"Masinloetav fail (Äriregister)"
"Inimloetav fail (Riigikogu, KOV)"

If we already have a decision on the format of the Help Button (do we, @kevincrepin?), let's add the Help Button with the content that Becca suggested. 

Moving back to Triage.

**Comment by loorm** [comment]
Triage 38.  Assigning this to Sepp. Prioritizing.

**Comment by seppviljar** [comment]
Will take it but I'll need contributor rights in github, being project memeber is not enough

**Comment by BeccaMelhuish** [comment]
@tiblu sorry if a bit late, but can I just check what the English versions says - is it just "Machine-readable file" and "Human-readable file" or does it have the "(Äriregister)" and "(Riigikogu, KOV)" in there too. (Can't view this myself). 

As if no brackets, I'd suggest adding them into the English ones too, as it will only be Estonian people who ever (at least currently) view this as they're the only ones using e-ID. But from what I understand many Estonian users use the platform in English.   

(As previously mentioned I'm not keen on the machine/human-readable bit in terms of clearing up confusion for the user, but I think the brackets should help).

**Comment by BeccaMelhuish** [comment]
And then for the new text for the Help icon once we go forwards with that, it would now be (if KOV means local gov?):

Submitting to the Äriregister? Choose “Machine-readable file”.

Submitting to the Riigikogu or KOV? Choose “Human-readable file”.

The "human-readable" option includes a CSV file. This is data presented in a tabular format which can be opened in any spreadsheet programme, such as Excel or Google Sheets.

Need more guidance? View Q4 and Q5 in our FAQs. 
[English FAQs](https://citizenos.com/faq/). [Estonian KKKs](https://citizenos.com/et/kkk/).

**Comment by seppviljar** [comment]
Hey. Current version has additional samples in Estonian, only. The source, English, is plain "Human-readable file" / "Machine-readable file"

**Comment by BeccaMelhuish** [comment]
OK thanks Sepp! 

Then yes my suggestion would be to include "(Äriregister)" and "(Riigikogu, KOV)" in all languages, but not to translate those particular words. As when we discussed this initially (me, Anett, Ilmar, Mikk) we decided that keeping them in Estonian would be clearest, as it is currently only Estonians/Estonian residents who'll be reading this. (No-one else would be downloading e-signature files).
